src/lib.rs

```diff
@@ -175,6 +179,10 @@ impl HSpanContext {
     }
 }
 
+pub fn null_tracer() -> Tracer {
+    Tracer::new(NullSampler).0
+}
+
 /// TODO: use lazy_static / thread_local singleton Tracer
 fn noop(name: String) -> HSpan {
     Tracer::new(NullSampler).0.span(name).start().into()
```

Comment:
Might as well use the `null_tracer` function you just created. Also according to this comment, it would be cool to make it a lazy_static singleton since this is stateless and every `noop` can share the same null tracer